### PR TITLE
don't wait for distinctInstance services

### PR DESCRIPTION
**Pull request by rgarcia**

### CODE REVIEW

File: src/rollover.py [hunk 1]
```diff
@@ -257,6 +257,9 @@ def wait_for_all_services(ecs_client, services_on_instance, service_events):
     failed = []
     for service_id in services_on_instance:
         last_event = service_events[service_id][-1]
+        if len(service_descriptions[service_id]["placementConstraints"]) == 1 and service_descriptions[service_id]["placementConstraints"][0]["type"] == "distinctInstance":
+            # skip distinct instance services
```
Comment (nathanleiby) [inline src/rollover.py]: Maybe print a warning that we're skipping this service?

**Comment by nathanleiby** [comment]
lgtm

**Comment by nathanleiby** [comment]
cc @burnsed

File: src/rollover.py (hunk 1 above)
Comment (rgarcia) [reply src/rollover.py]: 4ef3f6a

**Pull request merged**